earth_enterprise/src/fusion/mtjtools/scroll/zoom.h

```diff
@@ -417,7 +417,7 @@ class Image
   {
 #ifdef MALLOG
     log = fopen("c:/mallog.txt", "a"); // XXX
-    fprintf(log, "\nImage::Image(%d, %d, 0x%08x ... zoom)\n\n", x, y, source);
+    fprintf(log, "\nImage::Image(%d, %d, 0x%08x ... zoom)\n\n", x, y, static_cast<unsigned int>(source));
```

Comment:
This one confuses me. How are we able to cast an `Image &` to an `unsigned int`?

Reply:
I did some research on this and there are some compile failures if you `#define MALLOG` at the top of this file. I'd be fine with simply removing all the MALLOG sections from this file. That would fix the issue and we won't loose anything we're using or we can't recreate if we need to.